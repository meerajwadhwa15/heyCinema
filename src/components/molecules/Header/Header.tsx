import React, { FunctionComponent } from 'react';

export type Props = {};

const Header: FunctionComponent<Props> = props => (
  <header className="header">
    <h1>
      <div>hey</div>
      <div>cinema</div>
    </h1>
  </header>
);

export default Header;
