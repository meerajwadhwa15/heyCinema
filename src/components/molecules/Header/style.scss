header.header {
  background: $secondaryBackgroundColor;
  display: flex;
  padding: 10px;
  margin-bottom: 20px;

  h1 > div {
    color: $primaryColor;

    &:last-child {
      color: $secondaryColor;
    }
  }
}
