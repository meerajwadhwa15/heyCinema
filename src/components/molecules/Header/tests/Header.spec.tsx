import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import Header, { Props } from '../Header';

const props: Props = {};

describe('Header Enzyme testing', () => {
  test('renders without crashing', () => {
    shallow(<Header {...props} />);
  });
});

describe('Header Snapshot testing', () => {
  test('Matches the snapshot', () => {
    const component = create(<Header {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
