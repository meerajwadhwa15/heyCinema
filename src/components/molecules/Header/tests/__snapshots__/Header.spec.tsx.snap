// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot testing Matches the snapshot 1`] = `
<header
  className="header"
>
  <h1>
    <div>
      hey
    </div>
    <div>
      cinema
    </div>
  </h1>
</header>
`;
